src/microfrontends/assetManifestUtils.ts

```diff
@@ -0,0 +1,37 @@
+import { ManifestObject } from '@navikt/navspa/dist/async/async-navspa';
```

Comment:
Hva er formålet med denne fila? :)

Reply:
@Brunarspunar , det brukes til config for NavSpa biblioteket :)